server/app/controllers/fusor/api/v21/deployments_controller.rb

```diff
@@ -160,11 +160,16 @@ def mount_storage(address, path, type)
 
       raise 'Unable to mount NFS share at specified mount point' unless status == 0
 
+      files = Dir["/tmp/fusor-test-mount-#{deployment_id}/*"]
+
       stats = Sys::Filesystem.stat("/tmp/fusor-test-mount-#{deployment_id}")
       mb_available = stats.block_size * stats.blocks_available / 1024 / 1024
 
       Utils::Fusor::CommandUtils.run_command("sudo safe-umount.sh #{deployment_id}")
-      return mb_available
+      return {
+        :mb_available => mb_available,
+        :is_empty => files.size == 0
```

Comment:
This is wrapped in a rescue that will return a 200 mounted => false, so it's gracefully handled. I forgot to add `is_emtpy`, will update that, but I think this should be covered. (Also, I'm not sure it can return nil? http://ruby-doc.org/core-2.2.3/Dir.html#method-c-glob)